src/nomad_material_processing/general.py

```diff
@@ -0,0 +1,1336 @@
+        type=float,
+        description='Area of the lateral surface.',
+        a_eln=ELNAnnotation(
+            component='NumberEditQuantity',
+            defaultDisplayUnit='millimeter ** 2',
+        ),
+        unit='meter ** 2',
+    )
+
+
+class CylinderSector(Cylinder):
+    central_angle = Quantity(
+        type=float,
+        description="""The angle that defines the portion of the cylinder.
+        This angle is taken at the center of the base circle
+        and extends to the arc that defines the cylindrical sector.""",
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.NumberEditQuantity,
+        ),
+        unit='degree',
+    )
+
+
+class IrregularParallelSurfaces(Geometry):
+    """
+    A shape that does not fit into any of the other geometry classes.
+    """
+
+    m_def = Section()
+    height = Quantity(
+        type=float,
+        description='The z dimension of the irregular shape.',
+        a_eln=ELNAnnotation(
+            component='NumberEditQuantity',
+            defaultDisplayUnit='nanometer',
+        ),
+        label='Height (z)',
+        unit='meter',
+    )
+
+
+class Miscut(ArchiveSection):
+    """
+    The miscut in a crystalline substrate refers to
+    the intentional deviation from a specific crystallographic orientation,
+    commonly expressed as the angular displacement of a crystal plane.
+
+    It can be either directed along one of the crystallographic axes or in a specific
+    direction in the crystal defined by a main direction and the perpendicular one.
+    """
+
+    angle = Quantity(
+        type=float,
+        description="""
+        The angular displacement offset toward the crystallographic
+        orientation of the substrate.
+        """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.NumberEditQuantity,
+            defaultDisplayUnit='deg',
+            label='Miscut Angle',
+        ),
+        unit='deg',
+    )
+    angle_deviation = Quantity(
+        type=float,
+        description='The ± deviation of the angular displacement offset.',
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.NumberEditQuantity,
+            defaultDisplayUnit='deg',
+            label='± Miscut Angle Deviation',
+        ),
+        unit='deg',
+    )
+    hkl_absolut_orientation = Quantity(
+        type=str,
+        description="""Direction where the miscut goes toward.
+        Here the absolut Miller indices, (hkl), denote a normal to the planes
+        in the basis of the primitive reciprocal lattice vectors. """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.StringEditQuantity,
+            label='Miscut Absolut Orientation (hkl)',
+        ),
+    )
+    hkl_relative_orientation = Quantity(
+        type=str,
+        description="""Direction where the miscut goes toward. It uses the direct
+        lattice basis instead of the reciprocal lattice. Note that [hkl] is not
```

Comment:
> Note that [hkl]

I think you're looking for `{hkl}`... They correspond to the set of `(hkl)` after applying the symmetry operations (of the conventional cell).

Note that you can also indicate directions that using `[hkl]` and direction families with `<hkl>`.

Reply:
{hkl} represent a set of equivalent fsmilies, like 001, 010, 100 in cubic systems.

[hkl] is different, it is the family of planes in the direct space

Reply:
@ndaelman-hu https://en.wikipedia.org/wiki/Miller_index

check it out